Solution1/ConsoleApp/Services/UserCommunicateService.cs

```diff
@@ -3,70 +3,99 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using BusinessLayer.Extensions;
-using BusinessLayer.Services.Abstract;
+using ConsoleApp.Command;
+using ConsoleApp.Command.Abstract;
+using ConsoleApp.Extensions;
 using ConsoleApp.Services.Abstract;
+using FluentValidation;
 using Microsoft.Extensions.Logging;
 
 namespace ConsoleApp.Services
 {
     public class UserCommunicateService : IUserCommunicateService
     {
         private readonly ILogger _logger;
-        private readonly IWeatherServiсe _weatherServiсe;
+        private readonly IInvoker _invoker;
+        private readonly IPerformerCommandsService _performerCommandsService;
 
-        public UserCommunicateService(ILogger logger, IWeatherServiсe weatherService)
+        public UserCommunicateService(ILogger logger, IInvoker invoker, IPerformerCommandsService performerCommandsService)
         {
             _logger = logger;
-            _weatherServiсe = weatherService;
+            _invoker = invoker;
+            _performerCommandsService = performerCommandsService;
         }
 
         public async Task<bool> CommunicateAsync()
         {
-            Console.WriteLine("Please, enter city name:");
+            Console.WriteLine("Select menu item:");
+            Console.WriteLine("0 - Exit");
+            Console.WriteLine("1 - Get currently weather");
+            Console.WriteLine("2 - Get weather for a period of time");
 
-            var cityName = Console.ReadLine();
+            bool parseResult = int.TryParse(Console.ReadLine(), out var pointMenu);
```

Comment:
var, boolean vars are named https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
+